### PR TITLE
refactor Ansible files

### PR DESCRIPTION
**Pull request by hvanz**
The goal of this PR is to simplify the scripts by adopting some of Ansible best practices. In particular: 
- put many common settings in `ansible.cfg`,
- organize files in a predefined [directory layout](https://docs.ansible.com/ansible/2.9/user_guide/playbooks_best_practices.html#directory-layout), so we can reuse common variables and tasks. Common tasks are now in files under `ansible/tasks` and common variables, in `ansibles/group_vars/all.yaml`, which by defaults is included in all playbooks.

### CODE REVIEW

File: ansible/testapp-update.yaml [hunk 1]
```diff
@@ -1,26 +1,12 @@
 - name: update testapp
-  hosts: validators,loadrunners,ephemeral
```
Comment (hvanz) [inline ansible/testapp-update.yaml:2]: The only relevant task in this playbook for loadrunners is `tasks/testapp-clone-repo.yaml`, which is already included `loader-init.yaml`.

**Comment by greg-szabo** [comment]
We should also disable gathering facts in playbooks that don't use it.

**Review by greg-szabo — APPROVED** [review]
The ultimate solution would be to minimize the need for ansible. Unfortunately, this is not completely possible as we need to ship files to/from the hosts.

I'd love to see a remote docker-compose solution to manage the hosts services, instead of systems services. Maybe on a deep-flow week we can play with that.

**Comment by hvanz** [comment]
I agree @greg-szabo. I did this refactoring while trying to understand the scripts. In the long run I hope this simplification of the code will help us to move away from this repo.

**Pull request merged**